### PR TITLE
Add seeNumberOfVisibleElements

### PR DESCRIPTION
**Pull request by elarouche**
Give the ability to not only see how many times an element appears in the DOM but how many times an element is actually visible.

Had to build this function as a custom helper for my project. I believe this should be implemented. Very useful, especially when writing acceptance tests.

### CODE REVIEW

**Comment by elarouche** [comment]
In other words, it is a combination of `seeElement` and `seeNumberOfElements` from the WebDriverIO helper

File: lib/helper/WebDriverIO.js [hunk 1]
```diff
@@ -757,6 +757,18 @@ class WebDriverIO extends Helper {
...
+  
+  /**
+   * asserts that an element is visible a given number of times
+   * Element is located by CSS or XPath.
+   */
+  seeNumberOfVisibleElements(selector, num) {
+    return this.browser.isVisible(withStrictLocator(selector))
+      .then(function(res) {
+        res = res.filter((val) => val == true);
+        return assert.equal(res.length, num);
```
Comment (DavertMik) [inline lib/helper/WebDriverIO.js]: please add assertion failure message specific for this case

File: lib/helper/WebDriverIO.js [hunk 2]
```diff
@@ -757,6 +757,18 @@ class WebDriverIO extends Helper {
         return assert.equal(res.value.length, num);
       });
   }
+  
+  /**
+   * asserts that an element is visible a given number of times
```
Comment (DavertMik) [inline lib/helper/WebDriverIO.js]: Please add longer documentation with code example; similar to other methods.

**Comment by elarouche** [comment]
Not a 100% sure about that `add assertion failure message` commit. Went with ``return truth(`elements of ${locator}`, 'to be seen').assert.equal(res.length, num);``, is that the best way to go about it?

**Comment by DavertMik** [comment]
Yes, looks good now

**Pull request merged**